pandas/core/strings/accessor.py

```diff
@@ -1374,6 +1378,9 @@ def replace(self, pat, repl, n=-1, case=None, flags=0, regex=None):
         elif callable(repl):
             raise ValueError("Cannot use a callable replacement when regex=False")
 
+        if case is None:
+            case = True
```

Comment:
changed so that case is not optional on array methods. all validation is done here.